### PR TITLE
feat: allow adding external links to shortcuts

### PR DESCRIPTION
**Pull request by drodil**
## Hey, I just made a Pull Request!

This is behind a new property `allowExternalLinks` to make sure the application developer knows what he is doing. Only http(s) links are allowed.

image

image

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/shortcuts/src/ShortcutItem.tsx [hunk 1]
```diff
@@ -58,9 +58,10 @@ const getIconText = (title: string) =>
 type Props = {
   shortcut: Shortcut;
   api: ShortcutApi;
+  allowExternalLinks?: boolean;
 };
 
-export const ShortcutItem = ({ shortcut, api }: Props) => {
+export const ShortcutItem = ({ shortcut, api, allowExternalLinks }: Props) => {
```
Comment (Rugvip) [inline plugins/shortcuts/src/ShortcutItem.tsx:64]: Will the existing `SidebarItem` handle external links correctly? 🤔

File: .changeset/strong-crews-repeat.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-shortcuts': minor
```
Comment (Rugvip) [inline .changeset/strong-crews-repeat.md:2]:
```suggestion
'@backstage/plugin-shortcuts': patch
```

Still on 0.x

**Review by Rugvip — CHANGES_REQUESTED** [review]
Thank you! 👍

File: plugins/shortcuts/src/ShortcutItem.tsx (hunk 1 above)
Comment (drodil) [reply plugins/shortcuts/src/ShortcutItem.tsx:64]: Yeah, you can test it in the Uffizzi environment

**Pull request merged**